inst/shiny/modules/tab_nca/setup/data_imputation.R

```diff
@@ -0,0 +1,124 @@
+data_imputation_ui <- function(id) {
+  ns <- NS(id)
+
+  tagList(
+    input_switch(
+      id = ns("should_impute_c0"),
+      label = "Impute Concentration",
+      value = TRUE
+    ),
+    br(),
+    helpText(HTML(paste(
+      "Imputes a start-of-interval concentration to calculate non-observational parameters:",
+      "- If DOSNO = 1 & IV bolus: C0 = 0",
+      "- If DOSNO > 1 & not IV bolus: C0 = predose",
+      "- If IV bolus & monoexponential data: logslope",
+      "- If IV bolus & not monoexponential data: C0 = C1",
+      sep = "<br>"
+    ))),
+    br(),
+    br(),
+    selectInput(
+      ns("select_blq_strategy"),
+      "Select BLQ Imputation Strategy",
+      choices = c(
+        "tmax based imputation",
+        "Positional BLQ imputation",
+        "Set value for all BLQ"
```

Comment:
Nitpick: formatting.
```suggestion
        "Tmax based imputation",
        "Positional BLQ imputation",
        "Set value for all BLQ"
```

Reply:
nice catch